.github/workflows/build-release.yml

```diff
@@ -190,6 +190,7 @@ jobs:
     uses: ./.github/workflows/publish-extension.yml
     with:
       prerelease: ${{ github.event.inputs.prerelease }}
+      publish: ${{ github.event.inputs.draft == 'false' && github.event.inputs.prerelease == 'false' }}
```

Comment:
According to following codes, both `draft` and `prerelease` type are boolean not string :)

https://github.com/Samsung/ONE-vscode/blob/36f886e3b09048a3fc981550169a721edc2e4be8/.github/workflows/build-release.yml#L11

Reply:
The `github.event.inputs.<name>` always string even its type is boolean.

> Note: The workflow will also receive the inputs in the github.event.inputs context. The information in the inputs context and github.event.inputs context is identical except that the inputs context preserves Boolean values as Booleans instead of converting them to strings.

from https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
The latest guideline suggests to use `input` context instead of `github.event.inputs`. So I will change it.